packages/rx_bloc_cli/mason_templates/bricks/lib_pin_code/__brick__/lib/lib_pin_code/views/verify_pin_code_page.dart

```diff
@@ -87,10 +91,13 @@ class _VerifyPinCodePageState extends State<VerifyPinCodePage> {
   Future<void> _isAuthenticatedWithBiometrics(
     bool isAuthWithBiometrics,
   ) async {
-    if (isAuthWithBiometrics && widget.pinCodeArguments.isSessionTimeout) {
+    if (isAuthWithBiometrics) {
```

Comment:
Any reason to have a nested if statement here instead of how it was?

Reply:
No, this is returned as it was